src/app/shared/models/dot-push-publish-dialog-data/dot-push-publish-dialog-data.model.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Data that came from legacy PushPublish Dialog
+ * @export
+ * @interface DotPushPublishDialogData
+ */
+
+export interface DotPushPublishDialogData {
+    assetIdentifier: string;
+    title: string;
+    dateFilter?: boolean;
+    removeOnly?: boolean;
+    isBundle?: boolean;
+    restricted?: boolean;
+    cats?: boolean;
```

Comment:
cats?

Reply:
is the name used in the JSP currently, just replicating that in Angular side